tock/employees/views.py

```diff
@@ -51,6 +54,7 @@ def get_context_data(self, **kwargs):
         kwargs['username'] = self.kwargs['username']
         user = User.objects.get(username=kwargs['username'])
         context = super(UserFormView, self).get_context_data(**kwargs)
+        context.update(user_billing_context(user))
```

Comment:
Just wondering, should we consider a context processor and just do this everywhere? Is that overkill?

Reply:
I think _yes_ if/when we want to display utilization details on pages beyond this specific view.